msgraph/valuetypes.go

```diff
@@ -49,6 +49,29 @@ const (
 	BodyTypeHtml BodyType = "html"
 )
 
+type ExtensionSchemaTargetType string
+
+const (
+	ExtensionSchemaTargetTypeContact      ExtensionSchemaTargetType = "Contact"
+	ExtensionSchemaTargetTypeDevice       ExtensionSchemaTargetType = "Device"
+	ExtensionSchemaTargetTypeEvent        ExtensionSchemaTargetType = "Event"
+	ExtensionSchemaTargetTypePost         ExtensionSchemaTargetType = "Post"
+	ExtensionSchemaTargetTypeGroup        ExtensionSchemaTargetType = "Group"
+	ExtensionSchemaTargetTypeMessage      ExtensionSchemaTargetType = "Message"
+	ExtensionSchemaTargetTypeOrganization ExtensionSchemaTargetType = "Organization"
+	ExtensionSchemaTargetTypeUser         ExtensionSchemaTargetType = "User"
```

Comment:
```suggestion
	ExtensionSchemaTargetTypeAdministrativeUnit ExtensionSchemaTargetType = "AdministrativeUnit"
	ExtensionSchemaTargetTypeContact            ExtensionSchemaTargetType = "Contact"
	ExtensionSchemaTargetTypeDevice             ExtensionSchemaTargetType = "Device"
	ExtensionSchemaTargetTypeEvent              ExtensionSchemaTargetType = "Event"
	ExtensionSchemaTargetTypePost               ExtensionSchemaTargetType = "Post"
	ExtensionSchemaTargetTypeGroup              ExtensionSchemaTargetType = "Group"
	ExtensionSchemaTargetTypeMessage            ExtensionSchemaTargetType = "Message"
	ExtensionSchemaTargetTypeOrganization       ExtensionSchemaTargetType = "Organization"
	ExtensionSchemaTargetTypeUser               ExtensionSchemaTargetType = "User"
```

Reply:
Hey @manicminer sorry for slow response, i was away. 

So i dont believe `AdministrativeUnit` is supported at least for now by Schema extensions. I may be wrong, im basing my understanding of this from this doc: https://docs.microsoft.com/en-us/graph/api/resources/schemaextension?view=graph-rest-1.0 which details:
```Set of Microsoft Graph types (that can support extensions) that the schema extension can be applied to. Select from contact, device, event, group, message, organization, post, or user.```

Reply:
Ah right, we're actually using the beta endpoint in this client which is where I got the list of values from (https://docs.microsoft.com/en-us/graph/api/resources/schemaextension?view=graph-rest-beta). We could add the value, or switch to the v1.0 endpoint?